pandas/tests/frame/test_apply.py

```diff
@@ -553,6 +555,26 @@ def test_apply_non_numpy_dtype(self):
         result = df.apply(lambda x: x)
         assert_frame_equal(result, df)
 
+    @pytest.mark.parametrize('time_in', ['22:05 UTC+1', '22:05'])
+    @pytest.mark.parametrize("test_input", [
+        'string text',
+        parse('22:05'),
+        parse('12:13 UTC+1'),
+        parse('15:56 UTC+2'),
+        42,
+        3.14159, ])
+    def test_gh_19359(self, time_in, test_input):
```

Comment:
add the gh issue number as a comment. you can rename the test name to something more informative